packages/gatsby/content/features/plugnplay.md

```diff
@@ -38,29 +38,56 @@ This approach has various benefits:
 
 - Faster application startup, because the Node resolution doesn't have to iterate over the filesystem hierarchy nearly as much as before (and soon won't have to do it at all!).
 
+## PnP `loose` mode
+
+PnP `strict` mode breaks packages that rely on hoisting.
```

Comment:
```suggestion
Because the hoisting heuristics aren't standardized and predictable, PnP operating under strict mode will prevent packages to require dependencies that they don't explicitly list (even if one of their others dependencies happens to depend on it). This may cause issues with some packages.
```